src/components/ClipboardButton.js

```diff
@@ -8,7 +8,7 @@ import { icon } from 'src/components/icons';
 import { withErrorReporting } from 'src/libs/error';
 import * as Utils from 'src/libs/utils';
 
-export const ClipboardButton = ({ text, onClick, children, ...props }) => {
+export const ClipboardButton = ({ text, onClick, children, iconSize, ...props }) => {
```

Comment:
The build failed because of a TypeScript error:
https://app.circleci.com/pipelines/github/DataBiosphere/terra-ui/19058/workflows/1f540f67-b695-4276-9bd1-0f5e6105040a/jobs/69101

Unfortunately, since ClipboardButton is in a JavaScript file, TypeScript can't understand that `iconSize` is optional in ClipboardButton props. I think adding a default value should fix it.

```suggestion
export const ClipboardButton = ({ text, onClick, children, iconSize = undefined, ...props }) => {
```

Reply:
Ah, weird the local test didn't seem to catch this. Just pushed the change, thanks!

Reply:
Yeah, this doesn't change the behavior at all, so the tests would pass without this. It's the type check (`yarn run tsc`) that failed.

Reply:
Nice, thanks. Would it make sense to consider adding that check as part of the standard commit hooks for the repo?

Reply:
Possibly. But yeah, it takes ~10s to run.

Reply:
Yeah, might not be worth it... Thanks!